src/webview_provider/newLineagePanel.ts

```diff
@@ -639,14 +640,15 @@ export class NewLineagePanel implements LineagePanelView {
       if (this.cancellationTokenSource?.token.isCancellationRequested) {
         return { column_lineage: [] };
       }
+      const session_id = `${env.sessionId}-${targets[0]}-${targets[1]}`;
       const request = {
         model_dialect: modelDialect,
         model_info: modelInfos,
         upstream_expansion: upstreamExpansion,
         targets: targets.map((t) => ({ uniqueId: t[0], column_name: t[1] })),
         selected_column: selected_column!,
         parent_models,
-        session_id: sessionId,
+        session_id,
```

Comment:
should it be session_id: sessionId?

Reply:
same for parent_models?

Reply:
Yeah, indeed should use camelcase as it is a simple JS variable, not a API object.